ghcide/src/Development/IDE/LSP/HoverDefinition.hs

```diff
@@ -53,18 +55,6 @@ foundHover :: (Maybe Range, [T.Text]) -> Maybe Hover
 foundHover (mbRange, contents) =
   Just $ Hover (HoverContents $ MarkupContent MkMarkdown $ T.intercalate sectionSeparator contents) mbRange
 
-setIdeHandlers :: LSP.Handlers (ServerM c)
```

Comment:
This is necessary, since these IdeHandlers are wired into HLS. Otherwise, Hover will trigger a typecheck, consequentially typechecking the .cabal file.

Reply:
I mean: make a PR for removing the wired in handlers, then we can merge this one afterwards.

Reply:
sure, incoming in a couple of minutes